test/middleware.js

```diff
@@ -31,6 +33,45 @@ describe('Creating middleware', function() {
 
 });
 
+describe('Log messages', function() {
+  it('should use the default logger when none provided', function(done) {
+    var expected = '\u001b[90msource:\u001b[0m \u001b[36m' + index_scssFile + '\u001b[0m';
+    var bin = spawn('node', [fixture('example-server.js')]);
+
+    setTimeout(function() {
+      http.request({ method: 'GET', host: 'localhost', port: '8000', path: '/index.css' })
+          .end();
+    }, 500);
```

Comment:
This timeout is only required by Travis. Locally, making an HTTP request immediately after the spawn works in both macOS and windOS. We can also increase this test case timeout if it starts to fail, but lets monitor it for a while..